lxc/auth.go

```diff
@@ -1074,6 +1077,61 @@ func (c *cmdIdentityEdit) run(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+type cmdIdentityDelete struct {
+	global *cmdGlobal
+}
+
+func (c *cmdIdentityDelete) command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = usage("delete", i18n.G("[<remote>:]<authentication_method>/<name_or_identifier>"))
+	cmd.Aliases = []string{"rm"}
+	cmd.Short = i18n.G("Delete an identity")
+	cmd.Long = cli.FormatSection(i18n.G("Description"), i18n.G(
+		`Delete an identity`))
+	cmd.Example = cli.FormatSection("", `lxc auth identity delete oidc/jane.doe@example.com
+	Delete the OIDC identity with email address "jane.doe@example.com" in the default remote.
+
+lxc auth identity delete oidc/'Jane Doe'
+	Delete the OIDC identity with name "Jane Doe" in the default remote (there must be only one OIDC identity on the server with this name).
+
+lxc auth identity delete bramble-01:tls/12beaccbf9e7b7445185581b70099a5962c927e85006d5883856d909fe79f976
```

Comment:
bramble?

Reply:
Not sure what I was thinking here. Raspberry Pi's on the brain I think :laughing: What shall I use as an example remote name?

Reply:
Changed to "my-remote".